src/util/Constants.js

```diff
@@ -413,14 +416,15 @@ exports.SystemMessageTypes = exports.MessageTypes.filter(
 
 /**
  * <info>Bots cannot set a `CUSTOM` activity type, it is only for custom statuses received from users</info>
- * The type of an activity of a users presence, e.g. `PLAYING`. Here are the available types:
+ * The type of an activity of a users presence. Here are the available types:
```

Comment:
```suggestion
 * The type of activity of a user's presence. Here are the available types:
```

Reply:
I don’t agree with this because a presence can have multiple activities

Reply:
Edited that back to `an activity`.